### DISCUSSION TITLE
Could it support the new OpenTracing API

### DISCUSSION THREAD

**Discussion by dachuan9e** [apache/skywalking#7564] [Q&A]
Please answer these questions before submitting your issue.

- Why do you submit this issue?
- [x] Feature or performance improvement

___
### Requirement or improvement
- OpenTracing released and changed API from version 0.31.0.  It uses the old version 0.30.0 released in 2017 in Skywalking for OpenTracing suppport. The latest version 0.33.0 of OpenTracing is wildly used in many tracing components like Jaeger, Zipkin. Could it upgrade to support the newly OpenTracing.

**Comment by wu-sheng** [comment]
OpenTracing is not widely used. There is no PMC member thinking it is meaningful enough to take time. If you want to evaluate and share the contribution solution, we could follow.
Otherwise, I think there will be no progress.

**Reply by dachuan9e** [reply]
I need make a correction. I mean that if OpenTracing is used the higher version is used more than the lower. Most new components like Jaeger, Zipkin supported the newer version of OpenTracing. However, I agree that OpenTracing is not wildly used.

**Reply by wu-sheng** [reply]
We actually nearly don't need it. We have all auto instrumentation ways. 
So, there is no argument from me about shoud or shouldn't.
My point is no one willing to do. And in SkyWalking, no one willing means nothing will happen.

**Reply by dachuan9e** [reply]
That's true. Thanks for your answer.
